src/plan/join.rs

```diff
@@ -67,8 +67,7 @@ impl<P1: Implementable, P2: Implementable> Implementable for Join<P1, P2> {
                         .collect()
                 },
             );
-            // .inspect(|x| { println!("JOINED {:?}", x) })
```

Comment:
this seems to be `cargo fmt` at work? if so, please take it out of the PR
(we should've coordinated the cargo fmt thing better :P)

Reply:
Yep